### PR TITLE
MC-10744 (1): Add docs template to PR template

### PR DESCRIPTION
**Pull request by patrickspensieri**
#### Changes made
- create a template for our api-docs
- add the template to our PR template

#### Related PRs
- #133 
- #134

### CODE REVIEW

**Review by sweetsweetstefan — APPROVED** [review]
I don't have the full background on this, but I read the relevant context in the ticket and the file, and the proposed changes make sense to me.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,61 @@
...
+```js
+{
+  "taskId": "c50390c7-9d5b-4af4-a2da-e2a2678a83e8",
+  "taskStatus": "SUCCESS"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/releases/:id?operation=upgrade</code>
+
+Upgrade a release in a given [environment](#administration-environments)
```
Comment (lamminade) [inline .github/PULL_REQUEST_TEMPLATE.md:47]: no period on this sentence?

**Review by lamminade — APPROVED** [review]
looks good !

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (patrickspensieri) [reply .github/PULL_REQUEST_TEMPLATE.md:47]: FIxed [here](https://github.com/cloudops/cloudmc-api-docs/pull/134/commits/996dc0a649c40dabc549d50add9dc3ba75bebec9) and added periods to all section descriptions

**Pull request merged**